packages/react-magma-dom/src/components/Stepper/Stepper.tsx

```diff
@@ -0,0 +1,131 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+
+import { ThemeContext } from '../../theme/ThemeContext';
+import { ThemeInterface } from '../../theme/magma';
+import { useIsInverse } from '../../inverse';
+import { Step, StepProps, StepStatus } from './Step';
+import { Button } from '../Button';
+// import { I18nContext } from '../../i18n';
+
+/**
+ * @children required
+ */
+export interface StepperProps extends React.HTMLAttributes<HTMLDivElement> {
```

Comment:
I think the Stepper props should be:
- active / activeStep: index of the active step
- isLabelVisuallyHidden
- isSummaryVisuallyHidden
- breakpoint: there's a requirement around this, but I have some questions about how to use it / if it's still applicable with only a vertical stepper
- isInverse
- testId